aiidalab/utils.py

```diff
@@ -243,6 +243,14 @@ def run_pip_install(*args, python_bin=sys.executable):  # type: ignore
     )
 
 
+def run_reentry_scan():  # type: ignore
+    return subprocess.Popen(
```

Comment:
If we want to make sure this can be run, we need `reentry` installed explicitly https://pypi.org/project/reentry/, correct? So it should be a dependency?

Reply:
Moreover, instead of calling it with subprocess, I think it makes more sense to call it with API directly:

```python
from reentry import manager
manager.scan()
```

Not 100% sure this is exactly the `reentry scan`, you need have a double-check.

Reply:
> If we want to make sure this can be run, we need reentry installed explicitly https://pypi.org/project/reentry/, correct? So it should be a dependency?

Well, you are correct, but we do not want to add that as a dependency, because it is not needed in the new Docker stack. So we implicitly rely on reentry being installed in the old Docker stack. Not sure what else we can do here.

Reply:
Ah, I see your point, this makes sense. I think we shouldn't put complex in this backport support. I'll give it a test and have it.